esp_c3_m1_i_kit/pins_arduino.h

```diff
@@ -0,0 +1,40 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define EXTERNAL_NUM_INTERRUPTS 30
+#define NUM_DIGITAL_PINS        30
+#define NUM_ANALOG_INPUTS       6
+
+#define analogInputToDigitalPin(p)  (((p)<NUM_ANALOG_INPUTS)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<NUM_DIGITAL_PINS)?(p):-1)
+#define digitalPinHasPWM(p)         (p < EXTERNAL_NUM_INTERRUPTS)
+
+static const uint8_t LED_BUILTIN = 18;
+static const uint8_t LED_BUILTIN = 19;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+
+static const uint8_t LED_GREEN = 4;
+static const uint8_t LED_RED = 3;
+static const uint8_t LED_BLUE = 5;
+
+static const uint8_t TX = 18;
+static const uint8_t RX = 19;
+
+static const uint8_t SDA = 4;
+static const uint8_t SCL = 5;
+
+static const uint8_t SS    = 23;
+static const uint8_t MOSI  = 21;
+static const uint8_t MISO  = 22;
+static const uint8_t SCK   = 28;
```

Comment:
the above four pins are not recommended for use or do not exist on the ESP32-C3. These should be remapped to valid pins.

Reply:
@wmjohnanderson - suggestion:
```
static const uint8_t SS    = 5;
static const uint8_t MOSI  = 10;
static const uint8_t MISO  = 6;
static const uint8_t SCK   = 7;
```

Reply:
Fixed.